src/ch/imedias/rsccfx/view/RsccShowTokenView.java

```diff
@@ -64,10 +56,12 @@ private void layoutForm() {
 
   private void initFieldData() {
     //populate fields which require initial data
-
+    // FIXME: make sure only to put things here that have to do with populating initial data
```

Comment:
@Kennox make sure only to put things here that have to do with populating initial data